packages/components/src/color-picker/picker.tsx

```diff
@@ -19,6 +19,46 @@ export const Picker = ( { color, enableAlpha, onChange }: PickerProps ) => {
 		: RgbStringColorPicker;
 	const rgbColor = useMemo( () => color.toRgbString(), [ color ] );
 
+	useEffect( () => {
+		const iframe = document.querySelector(
+			'iframe'
+		) as HTMLIFrameElement | null;
+		const picker = document.querySelector( '.react-colorful__saturation' );
+
+		if ( ! picker || ! iframe ) {
+			return;
+		}
+
+		const restoreIframePointerEvents = () => {
+			iframe.style.pointerEvents = '';
+			window.removeEventListener(
+				'pointerup',
+				restoreIframePointerEvents
+			);
+		};
+		// iframe elements represent a separate window, and therefore any pointer
+		// event happening on top on an iframe is not registered correctly from the
+		// color picker. Disabling pointer events on the iframe prevents this
+		// issue from happening.
+		const disableIframePointerEvents = () => {
+			iframe.style.pointerEvents = 'none';
+			window.addEventListener( 'pointerup', restoreIframePointerEvents );
```

Comment:
Using `window` for the `pointerup` event because we'd need to make sure that pointer events on the iframe are restored whenever the pointer is released